dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_metrics/fat/src/com/ibm/websphere/microprofile/faulttolerance/metrics/fat/tests/MetricRemovalTest.java

```diff
@@ -104,6 +97,7 @@ public void metricRemovalTest() throws Exception {
      */
     private String getMetricsPage() throws IOException {
         HttpURLConnection con = HttpUtils.getHttpConnection(server, "metrics");
+        con.addRequestProperty("accept", "text/plain");
```

Comment:
Should not not be necessary any longer with Small Rye 5.0 RC3 and later.  I removed it in other places where we were setting it.

Reply:
The other branch that this is an alternative to does not have RC3 in its history.  That may explain why the test fails there.  The branch is based on release / integration from over a month ago.